server/api/openapi.yaml

```diff
@@ -19816,7 +19816,7 @@ components:
           type: string
           description: Name of application addressed by OaM request
         release-number:
-          pattern: "^([0-9]{1,2}).([0-9]{1,2}).([0-9]{1,2})$"
+          pattern: "^([0-9]{1,2})\\.([0-9]{1,2})\\.([0-9]{1,2})$"
```

Comment:
will it be pattern: "^([0-9]{1,2})\.([0-9]{1,2})\.([0-9]{1,2})$"
Kindly please have a check and let me know.

Reply:
This curl with release number 1;4;5 passes the unescaped dot regexp.

`curl -X POST "http://localhost:3004/v1/record-oam-request" -H  "accept: application/json" -H  "user: User Name" -H  "originator: Resolver" -H  "x-correlator: 550e8400-e29b-11d4-a716-446655440000" -H  "trace-indicator: 1.3.1" -H  "customer-journey: Unknown value" -H  "operation-key: dsa" -H  "Content-Type: application/json" -d "{\"application-name\":\"RegistryOffice\",\"release-number\":\"1;4;5\",\"method\":\"CONNECT\",\"resource\":\"foo\",\"stringified-body\":\"string\",\"response-code\":200,\"user-name\":\"string\",\"timestamp\":\"2023-05-25T19:20:24+00:00\"}"`

Reply:
Just noticed you suggested a different regexp. Please use " ` " to enclose the regexp in code, otherwise the slashes are eaten by markdown.

Reply:
But still no, one slash generates an error while doing npm run start.

```
OamLog/server/node_modules/openbackhaul-oas3-tools/node_modules/js-yaml/lib/loader.js:187
  throw generateError(state, message);
  ^
YAMLException: unknown escape sequence (19819:35)

 19816 |           type: string
 19817 |           description: Name of application addressed by OaM request
 19818 |         release-number:
 19819 |           pattern: "^([0-9]{1,2})\.([0-9]{1,2})\.([0-9]{1,2})$"
-------------------------------------------^
```

Reply:
@DanaSunal , Many thanks for the clarification. I got confused with the pattern we have in the spec yaml file and the one in the express framework(where we have to escape such escape sequence with another backslash).